tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "include": [
+    "js",
+    "typings"
```

Comment:
In `tsconfig.json` I write files to check: `js` is a directory with source files, `typings` is a directory with typings declarations.